File: presets/MidiKit/JavaScript/Delay NoteOn Ch1 for two 1500ms.js
/**
 * @target stoermelder MIDI-KIT
 * @engine QuickJs
 * @author stoermelder
 * @description Delays every Note-On message on channel 1 for 1500ms
 */

function onMidiMessage(midiInput, msg) {
    if (midi.isNoteOn(msg) && midi.getChannel(msg) === 1) {
        midiOut.sendAfterMs(msg, 1500);
    }
}

File: presets/MidiKit/Lua/Delay NoteOn Ch1 for two 1500ms.lua
--[[
@target stoermelder MIDI-KIT
@engine minilua@v1
@author stoermelder
@description Delays every Note-On message on channel 1 for 1500ms
--]]

midi.onMessage = function(midiPort, msg)
    if midi.isNoteOn(msg) and midi.getChannel(msg) == 1 then
        midiOut.sendAfterMs(msg, 1500)
    end
end